lib/podio/client.rb

```diff
@@ -72,6 +72,23 @@ def authenticate_with_credentials(username, password, offering_id=nil)
       @oauth_token
     end
 
+    def authenticate_with_second_factor(nonce, secondFactor, useRecoverycode)
+      if useRecoverycode == "false"
```

Comment:
variable `useRecoverycode` is binary or string ?

Reply:
It is sent as ActionController::Parameters. Boolean values come as string.